src/utils/twitterAPI/useTwitter.jsx

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+
+export const fetchTwitterUser = async (username, dispatch) => {
+  try {
+    const res = await axios.get(
+      `https://twitter-api-fetch-userdata.netlify.app/api/fetchUserData?username=${username}`
+    );
+    if (res)
```

Comment:
since this is in try block, so if condition wasn't needed

Reply:
Actually had to implement check if res contained data or not, have updated it in the net PR